### DISCUSSION TITLE
History for rar.rb keeps returning 404

### DISCUSSION THREAD

**Discussion by numbermaniac** [Homebrew/discussions#4333] [Casks]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

If you go to https://github.com/Homebrew/homebrew-cask/blob/master/Casks/rar.rb and click on "History" on the right side of the page, GitHub loads for a while then returns a 404 page.

I thought GitHub was broken but it's only happening for this particular cask - doing the same on any other cask (or any other file on GitHub) works just fine. Why is it broken on this one file?

### Relevant casks

https://github.com/Homebrew/homebrew-cask/blob/master/Casks/rar.rb

**Comment by SMillerDev** [comment]
No, that's definitely GitHub
